src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -83,11 +83,17 @@ public function set(string $key, mixed $value): void
     }
 
     /**
-     * Returns true if the parameter is defined.
+     * @param bool $acceptNull
```

Comment:
```suggestion
     * @param bool $allowNull
```
isn't the "allow" term more common for our codebase?